packages/composites/proof-of-reserves/src/utils/protocol.ts

```diff
@@ -41,11 +54,30 @@ export const runProtocolAdapter = async (
   return callAdapter(execute, context, next, '_onProtocol')
 }
 
-const listAdapter = (jobRunID: string, data: { addresses: string[] }) => {
+const listAdapter = (
+  jobRunID: string,
+  data: { addresses: AddressList; chainId?: string; network?: string },
```

Comment:
If using the suggested change above, we get:

```suggestion
  data: AddressList,
```